b2sdk/utils.py

```diff
@@ -21,6 +21,10 @@
 
 from logfury.v0_1 import DefaultTraceAbstractMeta, DefaultTraceMeta, limit_trace_arguments, disable_trace, trace_call
 
+# File info header prefix
+FILE_INFO_HEADER_PREFIX = 'X-Bz-Info-'
+FILE_INFO_HEADER_PREFIX_LOWER = FILE_INFO_HEADER_PREFIX.lower()
+
```

Comment:
I don't think we should store it here - I believe most of this http stuff is currently in raw_api

Reply:
It’s here and not in raw_api to avoid circular imports